src/types/index.ts

```diff
@@ -948,17 +948,15 @@ export enum SigningScheme {
    * For SingleKey ecdsa
    */
   SingleKey = 2,
+
+  MultiKey = 3,
 }
 
 export enum SigningSchemeInput {
   /**
    * For Ed25519PublicKey
    */
   Ed25519 = 0,
-  /**
-   * For MultiEd25519PublicKey
-   */
-  MultiEd25519 = 1,
```

Comment:
We will have to provide some way to handle this old key type for compatibility reasons, let's circle back on it though

Reply:
This is removed from `SigningSchemeInput` but it still in `SigningScheme`. This is because you should pass in `MultiEd25519` scheme when creating an Account